src/main/java/gov/nasa/pds/registry/common/util/ExceptionUtils.java

```diff
@@ -6,24 +6,23 @@
  * 
  * @author karpenko
  */
-public class ExceptionUtils
-{
-    /**
-     * Extract original exception message from a stack trace. 
-     * @param ex Exception object
-     * @return Original error message
-     */
-    public static String getMessage(Exception ex)
-    {
-        if(ex == null) return "";
```

Comment:
I removed this line because it is inane. The routine is meant to be part of an try/catch where there must be an exception. If ex is somehow null, then a NullPointer should be thrown and person calling it with null should be taken behind the chemical shed.